resolwe_bio/management/commands/generate_samples.py

```diff
@@ -264,9 +271,9 @@ def create_data(self, reads_name='seq_reads', annotated=False, rseed=None):
             sample.descriptor = self.generate_sample_desciptor(d.name)
             sample.presample = False
             sample.save()
-            print "Created sample: {} (id={})".format(sample.name, sample.id)
+            self.stdout.write("Created sample: {} (id={})".format(sample.name, sample.id))
```

Comment:
Changed by following Django docs. Output can now be tested and compared by call_command() stringIO redirection.